depends/packages/native_ds_store.mk

```diff
@@ -1,10 +1,9 @@
 package=native_ds_store
-$(package)_version=1.1.2
+$(package)_version=67e238d3dc8427891d2c1c7f1d1fa6194c44a5a6
```

Comment:
Any reason why we should use a specific commit rather than `v1.3.0`?

Reply:
Good call. When I went looking in the repo, I assumed that [1.2.0](https://github.com/al45tair/ds_store/commit/f8a5f7f13eb345b0f9f7d2f0a1058b86aacd0774) was the latest release. I'd missed https://github.com/al45tair/ds_store/commit/aa82c93a272c27a528c903f2525c24b69ac54cff as it's not actually in the master branch.